### PR TITLE
Update info for the loadbot

### PR DESCRIPTION
**Pull request by zivkovicmilos**
# Description

This PR adds updated info for the loadbot docs.

### CODE REVIEW

File: docs/additional-features/stress-testing.md [hunk 1]
```diff
@@ -9,13 +9,14 @@ This guide assumes that:
 
 - You have a working polygon-sdk network up and running (using IBFT or dev)
 - Both your JSON-RPC and gRPC endpoints are reachable
```
Comment (munna0908) [inline docs/additional-features/stress-testing.md:11]: @zivkovicmilos I guess we dont need gRPC.

**Review by munna0908 — CHANGES_REQUESTED** [review]

File: docs/additional-features/stress-testing.md (hunk 1 above)
Comment (zivkovicmilos) [reply docs/additional-features/stress-testing.md]: Good catch, we relied on gRPC in an earlier version of the implementation 💯 

Fixed 👍 

c64b209

**Review by Kourin1996 — APPROVED** [review]
Looks good 👍

**Review by PtitLuca — APPROVED** [review]
LGTM !

**Pull request merged**